CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Chef Push Client Changes
 
+## 2.1.0
+
+* Fixed heartbeat buffering issue that can cause trauma on the push-jobs-server
+  after restart
+
```

Comment:
Might rephrase to "limited zeromq high water mark to prevent buffering of heartbeats, which cause packet floods when the push-jobs-server restarts"